buildpack/nginx.py

```diff
@@ -254,6 +257,8 @@ def get_access_restriction_locations():
             raise Exception(
                 "Can not override access restrictions on system path %s" % path
             )
+        if path in ["/files"]:
```

Comment:
Thanks for submitting this PR @markbastiaans ! The path location in this `if` statement should be `/file`.